src/components/DeploymentFlow/DeploymentFlow.component.jsx

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import _ from 'lodash';
-import ReactFlow, { Background, Handle } from 'react-flow-renderer';
-import LoadingBox from 'components/LoadingBox';
+import React, { useMemo } from 'react';
+import lodash from 'lodash';
```

Comment:
Qual foi motivo da mudança do import do lodash?
Temos outros arquivos com essa notação, talvez devêssemos trocar lá também, o que acham?

Reply:
O motivo foi um code smell que o sonar identificou. Em uma função desse componente apenas o segundo parâmetro é utilizado e a convenção em JS é usar **_** para parâmetros não usados em uma função. Isso gerou um code smell do sonar dizendo que a variável **_** já foi declarada no escopo superior.

Reply:
Daria também ao invés de importar o lodash inteiro a gente importar só a função que se for usar dele. Não tenho certeza, mas é possível que na hora do build o webpack coloque só as funções utilizadas no bundle final ao invés da lib inteira. Isso que o webpack faz se chama [Three Shaking](https://webpack.js.org/guides/tree-shaking/)